### ISSUE TITLE
Add reactchart

### ISSUE THREAD

**Issue by fundkis** [michaelrambeau/bestofjs#131]
Please add our open sourced charting library: https://fundkis.com/lab/react-chart
git: https://github.com/fundkis/reactchart
npm: https://www.npmjs.com/package/reactchart
Regards,

**Comment by michaelrambeau** [comment]
Hello @fundkis and @SylvainPlessis
Thank you for the recommendation.

The homepage mentions
> We depend only on React!

Not a big deal, but actually I think the project also depends in other libraries: `moment`, `underscore`, and `freezer`.

Talking about dependencies, maybe you could ditch `underscore` and use the JavaScript `Array.map()` instead of underscore `.map()` because it's almost 2018 😄  !

One more thing I wonder if there is a reason why you use `let` instead of `const` everywhere.
I guess you just replaced all `var` keywords from an old codebase.

**Comment by fk-hb** [comment]
Hello @michaelrambeau, sorry for the delay.
the affirmation "we depend only on React!" is meant for the graphical engine. Most react chart libraries are based on external charting engines (d3, chartjs...).
our library has its own charting engine.
`moment`, `underscore` and `freezer` are very lightweight libraries.
We can ditch `underscore` and even `moment` by rewriting some of the code. We can make optional `freezer` if we do even more rewriting but it's mandatory for some features: efficient realtime chart updates.
As you guessed `reactchart` was an internal project that we decided to open source. the dependency on moment and underscore and the systematic use of `let` instead of `const` are due to old codebase and coding styles.

**Comment by michaelrambeau** [comment]
Hello @fk-hb
Thank you for your feedback.

No worries, I just wanted to be sure that the project is still "actively maintained" for 2 reasons:

* there is no commit since February (which means a lot in the fast-paced React eco-system!), and these `let` variable everywehere seem so, how to say... a smell of old-code base :sweat_smile:.
* there are a lot of contenders in this category https://bestof.js.org/tags/chart

Just a quick tip: you could improve the README.md file by enabling "Syntax highlighting", in JavaScript snippets: you just have to specify the language name after the 3 backticks characters, like this:

```
```javascript
```

You know what I mean? Otherwise I could do a Pull Request, no problem :smile:

**Comment by SylvainPlessis** [comment]
Hello @michaelrambeau,

sorry for the (huge) delay from my part, this is a quite busy period...
Thanks for considering ```reactchart```, let me add a few informations. 
There are actually updates since February (even two new features), but the main problem with integrating those changes is that I wanted to update the ```README``` file before. And, well, as is sadly usual, documentation is not the highest priority...
As we update npm at the same time, I don't want to have too many versions, specially for ```README``` file updates (thanks for the tip, btw, it looks much much better).

So basically, considering this conversation, here's the TODO list I've derived for ```reactchart```:
  - finish the clean/update of the code (because I won't pretend that even though there have been uptades, it all looks nice and super up-to-date javascript),
  - finally finish the ```README``` file,
  - update everything,
  - come back for feedback

How does that look?

**Comment by michaelrambeau** [comment]
Hello @SylvainPlessis @fundkis and @fk-hb 
No worries, update everything when you have time, I just wanted to help you make the project more attractive.

Anyway `reactchart` is available on bestof.js.org, check the `Chart` tag: https://bestof.js.org/tags/chart

Let me know if you have any feedback about bestof.js.org, stars from the 3 of you are welcome too ;)
